projects/Mallard/Makefile

```diff
@@ -24,4 +24,11 @@ beta-android:
 	cd android && ./gradlew clean && cd ..
 	fastlane android beta
 
+release-android:
+	mkdir -p android/app/src/main/assets
+	yarn jetify
+	yarn bundle-android
```

Comment:
I don't think that step is required, I think this already runs as part of the gradle build, and I don't see how the grade build would even know in which folder to find the bundle built that way. I might be wrong.